apps/course-api/src/utils/redis_utils.ts

```diff
@@ -4,11 +4,16 @@ const redisClient = createClient({
 	url: process.env.REDIS_URL
 });
 
-process.on("exit", async () => {
-	console.log("exiting redis..");
-	await redisClient.disconnect();
-	console.log("redis exited");
-});
+if (process.env.APP_ENV === "beta" || process.env.APP_ENV === "prod") {
+	process.on("exit", async () => {
+		console.log("exiting redis..");
+		try {
+			redisClient.disconnect();
```

Comment:
```suggestion
			await redisClient.disconnect();
```
![image](https://github.com/stamford-syntax-club/course-compose/assets/24469918/512a8c03-8160-49c6-9ac0-62a438c896d8)